code/espurna/sensors/A02YYUSensor.h

```diff
@@ -0,0 +1,148 @@
+// -----------------------------------------------------------------------------
+// DYP A02YYUx Ultrasonic Distance Sensor
+// Copyright (C) 2024 by tsouthiv @ gmail dot com
+// -----------------------------------------------------------------------------
+
+// This should be also compatible with other similar sensors like JSN-SR04T
+// or AJ-SR04M in auto UART mode.
+
+#if SENSOR_SUPPORT && A02YYU_SUPPORT
+
+#pragma once
+
+#include "Arduino.h"
+#include "BaseSensor.h"
+
+#define _RX_LEN     16
+#define _RX_MASK    (_RX_LEN - 1)
```

Comment:
btw these should either be local to the class as 'constexpr static' or at least prefixed w/ A02YYU\_...

Reply:
Thanks for the comment, I've changed to static constexpr.